gradle.properties

```diff
@@ -14,3 +14,8 @@
 # org.gradle.parallel=true
 org.gradle.jvmargs=-Xmx2048M
 android.useAndroidX=true
+
+minSdkVersion=21
+targetSdkVersion=29
```

Comment:
Use SDK 28 here as 29 breaks **everything**!

Reply:
Termux is not Android 10 compatible application. See https://github.com/termux/termux-app/issues/1072.